libs/eo/origin-of-energy/shell/src/lib/eo-origin-of-energy-shell.component.ts

```diff
@@ -14,45 +14,183 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
-import { EoInlineMessageScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
-import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
-
+import { MatCardModule } from '@angular/material/card';
+import { EoMediaModule } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { EoPieChartScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
-  selector: 'eo-metering-points-shell',
+  selector: 'eo-origin-of-energy-shell',
   styles: [
     `
       :host {
         display: block;
+        max-width: 1040px; /* Magic UX number */
+      }
+
+      .chart-row {
+        display: flex;
+        margin-bottom: var(--watt-space-l);
+      }
+
+      .chart-card {
+        max-width: 584px; /* Magic UX number */
+        width: 100%;
+      }
+
+      .chart-box {
+        margin-top: var(--watt-space-m);
+        margin-right: calc(var(--watt-space-xl) - var(--watt-space-s));
+        margin-left: calc(var(--watt-space-xl) - var(--watt-space-s));
+      }
+
+      .chart-tips {
+        width: 360px; /* Magic UX number */
+        gap: var(--watt-space-l);
+        display: flex;
+        flex-direction: column;
+      }
+
+      .global-goals-box {
+        border: 1px solid #f9d557; /* Color not yet added to Watt */
```

Comment:
suggestion: Refer to Watt's *warning* color both here and in the SVG until colors in the Watt implementation are aligned with the specification in Figma. Otherwise, we have to come back here and fix it. We could also add a note to the enabler we have on aligning Energy Origin colors after the Watt implementation has been aligned with Storybook.

Reply:
Changed the variable for warning-yellow in the project SCSS variables.

Also fixed the usage of this variable in the inline-message component, because these two places were the only ones affected by this color change.